src/sent_name_client.cpp

```diff
@@ -0,0 +1,56 @@
+//
+// Created by hlf on 5/10/23.
+//
+// !src/client.cpp
+
+#include "ros/ros.h"
+#include "helifeng/SentName.h"
+#include <cstdlib>
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "sent_name_client");
+
+    /*if (argc != 2)
+    {
+        ROS_INFO("usage: add_two_ints_client X Y");
+        return 1;
+    }*/
+
+    ros::NodeHandle nh;
+    ros::ServiceClient client = nh.serviceClient<helifeng::SentName>("sent_name");
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]